docs/flow_reference.rst

```diff
@@ -0,0 +1,659 @@
+Flow Reference
+==============
+CumulusCI suite of standard flows can be grouped into various categories depending on their intended purpose.
+
+Continous Integration
+---------------------
+The suite of continuous integration flows are intended to execute in the context of a CI system.
+They typically involve the creation of an org to have tests (Apex and Robot) run against it.
+    
+``ci_beta``
+^^^^^^^^^^^
+This flow installs the latest beta version available and runs apex tests from the managed package. 
+
+.. code-block:: console
+
+    Flow Steps
```

Comment:
This header looks a bit odd here as part of the code block, but I can't quite see how the current code produces it. Was this render from an older version?

Reply:
Actually, is this whole file an older version?

Reply:
@davidmreed - This whole file was hand typed... well, vim macros helped me with the repetition :)